### ISSUE TITLE
TypeError: Cannot read property 'column_name' of null

### ISSUE THREAD

**Issue by shugydw** [hasura/graphql-engine#2480]
I am using Dockerized Hasura (`v1.0.0-beta.2`) and connected to an existing Postgres(`11.4`) in the same Docker network, using Docker Compose.

My database has 784 tables. Some of the tables are quite large.

Whenever I try to "Track" a table, it will be tracked and the following error happens:

```
TypeError: Cannot read property 'column_name' of null
    at main.js.gz:1
    at Array.map (<anonymous>)
    at main.js.gz:1
    at Array.forEach (<anonymous>)
    at n (main.js.gz:1)
    at ho (vendor.js.gz:1)
    at Xo (vendor.js.gz:1)
    at fs (vendor.js.gz:1)
    at ds (vendor.js.gz:1)
    at Zs (vendor.js.gz:1)
Oa @ vendor.js.gz:1
o.componentDidCatch.r.callback @ vendor.js.gz:1
Ea @ vendor.js.gz:1
wa @ vendor.js.gz:1
as @ vendor.js.gz:1
ls @ vendor.js.gz:1
(anonymous) @ vendor.js.gz:1
t.unstable_runWithPriority @ vendor.js.gz:1
Js @ vendor.js.gz:1
Zs @ vendor.js.gz:1
Ys @ vendor.js.gz:1
nu @ vendor.js.gz:1
Rr @ vendor.js.gz:1
```

The same happens when I click on "Relationships" and "Modify" for one of the tracked tables.

Once, the error is thrown, the Error page appears with a link to go back to "Home". When I click it, this error occurs:

```
Uncaught TypeError: Cannot read property 'contains' of null
    at Main.n (main.js.gz:1)
n @ main.js.gz:1
```

**Comment by shugydw** [comment]
Hi @rikinsk What do you need more info about?

**Comment by rikinsk** [comment]
@shugydw I couldn't figure out the exact source of the problem. I suspect this has to do with a particular table with some configuration that the console didn't expect. 

We had a similar issue reported recently where the user had no columns in a table and that caused a similar error as you have shared. That issue has been fixed. Just to ensure that it wasn't the same code block impacting you as well, can you confirm that you are still facing the issue?

If yes, could you specify how you ended up in this scenario. I am guessing you hit `Track All` on the console to track all the tables in one go and ended up with errors on all the tables?

**Comment by shugydw** [comment]
@rikinsk Interestingly, I no longer face the issue. Earlier, I tracked the tables individually. I just tested with the `Track All` feature and there is no issue.

I didn't change a bit in the database or Hasura, except that I have shutdown Docker [for Mac] yesterday and just started now. Weird but resolved, I think.

**Comment by rikinsk** [comment]
@shugydw That's good to hear. I think your issue was related with the fix put out in PR #2484.
